addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -4,6 +4,24 @@ import contentMenu from 'website.contentMenu';
 import weWidgets from 'wysiwyg.widgets';
 import {_t} from 'web.core';
 
+weWidgets.LinkPopoverWidget.include({
+    /**
+     * @override
+     */
+    start: function () {
+        // hide popover while typing on mega menu
+        if (this.$target.parents('.o_mega_menu').length) {
```

Comment:
Use `closest` instead of `parents`, otherwise it keeps searching in all ancestors until the body :wink: Without jquery it works too, and would not need the `.length`, but as you wish:

```javascript
if (this.$target[0].closest('.o_mega_menu')) {
```

In fact, you are in a widget, `this.$target` is a deprecated variable kept for compatibility. Use `this.$el` (or for the javascript solution `this.el`) (like for all widgets). Until owl... :)

```javascript
if (this.el.closest('.o_mega_menu')) {
```

Reply:
Good catch for the `closest` :+1: but this.$target is not the same as this.$el.
$el is the popover
$target is the anchor

Reply:
Oh, ok, my bad, just a coincidence that you used the same name as the compat variable in widgets then :+1: